clusters/hive/pools/hypershift/hypershift-ocp-4-8-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: hypershift-ocp-4-8-amd64-aws-us-east-1
+  namespace: hypershift-cluster-pool
+  labels:
+    product: ocp
+    version: "4.8"
+    architecture: amd64
+    cloud: aws
+    owner: hypershift
+    region: us-east-1
+spec:
+  baseDomain: hive.aws.ci.openshift.org
```

Comment:
This has to be a domain managed by your AWS account.

Reply:
Ah, I set up a new hosted zone for this and updated the value, thanks